src/test/java/org/elasticsearch/test/ElasticsearchIntegrationTest.java

```diff
@@ -1418,6 +1418,34 @@ public static void enableTranslogFlush(String index) {
         client().admin().indices().prepareUpdateSettings(index).setSettings(settings).get();
     }
 
+    /** Disables an index block for the specified index */
+    public static void disableIndexBlock(String index, String block) {
+        disableIndexBlock(client(), index, block);
+    }
+
+    /** Disables an index block for the specified index using a given {@link Client} */
+    public static void disableIndexBlock(Client client, String index, String block) {
```

Comment:
I added this method so that classes which extends `ElasticsearchSingleNodeTest` were also able to use this method.

Since it was the case in only 2 classes, I removed the method with the specific client and created two classes which extends `ElasticsearchIntegrationTest`: `IndicesStatsBlocksTests` and `IndicesSegmentsBlocksTests`.